e2e/test-app-v10-zone-v11/tsconfig.spec.json

```diff
@@ -6,9 +6,13 @@
     "module": "commonjs",
     "types": [
       "jest",
+      "jquery",
       "node"
     ]
   },
+  "files": [
+    "src/global.ts"
```

Comment:
Interesting, I didn't know "global" types could be added like this.

Reply:
Actually I don't like this way. I'd prefer more using `d.ts` instead, but ya it's possible indeed.

Reply:
Yeah, `d.ts` and explicit `import`s are definitively better